third_party/move/tools/move-cli/src/base/prove.rs

```diff
@@ -127,7 +128,10 @@ impl ProverTest {
             for_test: true,
             options: Some(ProverOptions::Options(std::mem::take(&mut self.options))),
         };
-        let res = cmd.execute(Some(pkg_path), move_package::BuildConfig::default());
+        let mut build_config = BuildConfig::default();
+        build_config.compiler_config.language_version = Some(LanguageVersion::latest_stable());
```

Comment:
Does this mean that there is now no way to run tests with older versions of the Move Language?  We may want to have that ability in case there are breaking changes to semantics at some point.

Reply:
Yeah, it is a good point although for this specific API, it is only used for testing the prover in `diem-framework` and `move-stdlib` in `third_party`, which actually do not use any new language features. So maybe I should remove the language version here? @brmataptos